src/partition.c

```diff
@@ -63,6 +67,9 @@
 #include "threadpool.h"
 #include "tools.h"
 
+#define IDX_MAX INT32_MAX
```

Comment:
These defines are not going to work with METIS builds. Not sure if scotch has a 32/64 bit build choice like
METIS, but these are defined as INT32_ and INT64_ as needed in `metis.h`. At least wrap them in an '#ifndef'.

Reply:
The `repartition_name[]` array needs a final entry for the SCOTCH repartitioning description.

Reply:
Just seeing `null` reported at present.